templates/settings.json.epp

```diff
@@ -22,20 +22,17 @@
   <%- } -%>
   "port" : <%= $etherpad::port -%>,
 
-  /*
   // Node native SSL support
-  // this is disabled by default
-  //
   // make sure to have the minimum and correct file access permissions set
   // so that the Etherpad server can access them
-
+  <%- if $etherpad::ssl == 'enable' { -%>
   "ssl" : {
-            "key"  : "/path-to-your/epl-server.key",
-            "cert" : "/path-to-your/epl-server.crt",
-            "ca": ["/path-to-your/epl-intermediate-cert1.crt", "/path-to-your/epl-intermediate-cert2.crt"]
+          "key"  : "<%= $etherpad::ssl_key -%>",
+          "cert" : "<%= $etherpad::ssl_cert -%>"
           },
-
-  */
+  <%- } else { -%>
+  // SSL module is not enable by default
```

Comment:
can you fix the small typo here? I guess you mean `enabled`?

Reply:
Yes, thanks. I fixed it.